docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -37,8 +44,13 @@ The ``distributor_config`` contains a ``remote`` section with the following sett
   The ssh user for remote server.
 
 ``ssh_identity_file``
-  Absolute path to the private key that will be used as the identity file for ssh. The key has to
-  be readable by user ``apache``.
+  Absolute path to the private key that will be used as identity file for ssh. The key must be
+  owned by user ``apache`` and must not be readable by other users. If key permissions are too lax,
+  the SSH application will refuse to use the key. Additionally, the system SELinux policy must
```

Comment:
Consider this to replace the rest after the word `Additionally`.

```
If SELinux is Enforcing, Pulp requires the key to be labeled with the ``httpd_sys_content_t`` SELinux context. This can be applied to the file with::
```